### PR TITLE
Add public function to get data by endpoint

### PR DESCRIPTION
**Pull request by AndreiEvstratenko**
# Pull Request

## Description

- add public function to get data by endpoint

I'm thinking about do we need to return `Dictionary` object like now or I should return `Data` object. @jexe @brentchow what do you think about that?

### Motivation and Context

- related https://app.zenhub.com/workspaces/engineering-5c520101e050c96f3e496a62/issues/svrf/svrf-wave/2773

### Types of changes

- [ ] Bug fix
- [x] New feature
- [ ] Maintenance

### Checklist

- [ ] Documentation
- [ ] Tests

### CODE REVIEW

File: SvrfSDK/Source/SvrfSDK.swift [hunk 1]
```diff
@@ -230,6 +230,24 @@ public class SvrfSDK: NSObject {
         }
     }
 
+    public static func getJSON(endPoint: String,
```
Comment (brentchow) [inline SvrfSDK/Source/SvrfSDK.swift]: Is this the best name for this function? I’m thinking something like:

`Svrf.request(endpoint: '/foo', /*...*/)`

Comment (AndreiEvstratenko) [reply SvrfSDK/Source/SvrfSDK.swift]: I like your variant. Implemented

File: SvrfSDK/Source/SvrfSDK.swift [hunk 2]
```diff
@@ -222,14 +228,50 @@ public class SvrfSDK: NSObject {
...
+                                          parameters: parameters,
+                                          onSuccess: { data in
+                                            svrfRequest.state = .completed
+                                            success(data)
+            }, onFailure: { error in
+                svrfRequest.state = .completed
+                if let failure = failure, let svrfError = error as? SvrfError {
+                    failure(svrfError)
+                }
+            })
```
Comment (brentchow) [inline SvrfSDK/Source/SvrfSDK.swift]: This should be aligned with `onSuccess`

File: SvrfSDK/Source/SvrfSDK.swift [hunk 3]
```diff
@@ -222,14 +228,50 @@ public class SvrfSDK: NSObject {
...
+                    }
                     failure(svrfError)
                 }
 
             })
         }
     }
 
+    /**
+    Requests data from the Svrf by endpoint.
```
[the remark this answers is not in the document]
Comment (AndreiEvstratenko) [reply SvrfSDK/Source/SvrfSDK.swift]: I added example of endpoint. User doesn't need to include `/v1/` and the whole url either

File: SvrfSDK/Source/SvrfSDK.swift [hunk 4]
```diff
@@ -222,14 +228,50 @@ public class SvrfSDK: NSObject {
...
+                    }
                     failure(svrfError)
                 }
 
             })
         }
     }
 
+    /**
+    Requests data from the Svrf by endpoint (f.e. "/vr/search")
```
Comment (brentchow) [inline SvrfSDK/Source/SvrfSDK.swift]:
```suggestion
    Requests data from the Svrf by endpoint (e.g. `/vr/search`)
```

e.g. not f.e.  It’s an abbreviation for the Latin phrase “exempli gratia” which means “for example”. So the user does not have to include the API version?

**Pull request merged**